pkg/compilers/osv/osv_aws.go

```diff
@@ -1,23 +1,36 @@
 package osv
 
 import (
+	"io/ioutil"
+	"os"
+
 	"github.com/emc-advanced-dev/pkg/errors"
 	"github.com/emc-advanced-dev/unik/pkg/types"
 )
 
-type OsvAwsCompiler struct {
-	OSvCompilerBase
-}
-
 const OSV_AWS_MEMORY = 1024
 
-func (osvCompiler *OsvAwsCompiler) CompileRawImage(params types.CompileImageParams) (_ *types.RawImage, err error) {
-	resultFile, err := osvCompiler.CreateImage(params, true)
+type AwsCompilerHelper struct {
+	CompilerHelperBase
```

Comment:
Why do you need to embed `CompilerHelperBase`? `AwsCompilerHelper` implements `UseEc2` function so there is no point in embedding it.